package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "lunr": "^2.3.7"
+  }
+}
```

Comment:
:thinking:

Reply:
I don't know what changed here, happened on it's own